.github/workflows/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM registry.fedoraproject.org/fedora:rawhide
+
+RUN dnf -y install fedpkg tito argparse-manpage bash-completion python3-devel python3-jsonschema python3-pytest
+RUN useradd -m -G mock -u 1001 user
+WORKDIR /home/user
+
+COPY mock/ /home/user/mock
+RUN chown -R user:user /home/user/mock
+USER user
+WORKDIR /home/user/mock/mock-core-configs
+RUN tito build --rpm
+WORKDIR /home/user/mock/mock
+RUN tito build --rpm
```

Comment:
Please use `tito build --test --rpm` here, otherwise you build an old (the latest "released") variant of the package.